src/actions/SupplierCreditActions.js

```diff
@@ -52,6 +60,8 @@ const create = () => (dispatch, getState) => {
         returnSum
       );
 
+      newSupplierCredit.category = category;
```

Comment:
When time permits, should probably be added as a param to `createRecord` or spread assignment.

Reply:
I disagree - I don't think passing everything in the creation which aren't always even needed is really the best idea